tests/update/patch_sle.pm

```diff
@@ -309,4 +309,10 @@ sub test_flags {
     return {milestone => 1, fatal => 1};
 }
 
+sub post_fail_hook {
+    my ($self) = @_;
+    type_string "power off\n";
+    $self->_cleanup() unless $self->{cleanup_called};
+}
```

Comment:
I don't think it's wise to merge with this change. If you just want to have it for temporary debugging then better don't create a PR at all but just reference a branch in your github fork for openqa-clone-custom-git-refspec . Also I don't think you want to completely overwrite the base class post_fail_hook. Better call `$self->SUPER::post_fail_hook;`

Reply:
Ok, this just for temporary debug, I will use a branch next time. If the current code won't block to create the qcow2, I will keep it. If not, I will update it with "$self->SUPER::post_fail_hook;". Thanks.